lib/facter/util/wmi.rb

```diff
@@ -2,7 +2,10 @@ module Facter::Util::WMI
   class << self
     def connect(uri = wmi_resource_uri)
       require 'win32ole'
-      WIN32OLE.connect(uri)
+      #WIN32OLE.connect(uri)
+      swbem = WIN32OLE.new("WbemScripting.SWbemLocator")
```

Comment:
Since `WIN32OLE` calls `OleInitialize` (with STA), does this actually work?

https://github.com/ruby/ruby/blob/ruby_2_1/ext/win32ole/win32ole.c#L1251
https://github.com/ruby/ruby/blob/ruby_2_1/ext/win32ole/win32ole.c#L2750